src/dotnet/commands/dotnet-test/LocalizableStrings.cs

```diff
@@ -52,5 +52,8 @@ internal class LocalizableStrings
                                         Logs are written to the provided file.";
 
         public const string CmdNoBuildDescription = @"Do not build project before testing.";
+
+        public const string RunSettingsArgsHelpText = @"Any extra commandline runsettings arguments that should be passed to vstest. See 'dotnet vstest --help' for available options.
+                                        Example: -- RunConfiguration.ResultsDirectory=""C:\users\user\desktop\Results Directory"" MSTest.DeploymentEnabled=false";
```

Comment:
This also seems nice-to-have. It's quite advanced of functionality and will be nicely documented by e.g. xunit folks when they help their customers use this feature. 

@nguerrera how are we doing with loc handoffs?